src/ert/scheduler/driver.py

```diff
@@ -46,6 +47,7 @@ async def submit(
           cwd: Working directory.
           name: Name of job as submitted to compute cluster
           num_cpu: Number of CPU-cores to allocate
+          realization_memory: Memory to reserve, in bytes. 0 means no allocation.
```

Comment:
This can potentially be misinterpreted as `no memory will be allocated`. Can we change this to indicate that the default will be used if nothing is specified?

Reply:
Yes, "reserve" is a wrong word here. Fixing.